app/services/currency_service.rb

```diff
@@ -0,0 +1,10 @@
+require 'rest-client'
+
+    class CurrencyService < ApplicationService
+        def initialize
+        end
+
+        def call

```

Comment:
Call needs some arguments. It should receive **date** 
```suggestion
        def call(date)

```